reference/tax_settings.v3.yml

```diff
@@ -146,7 +146,19 @@ components:
         should_subtract_store_tax:
           default: true
           type: boolean
-          description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.    
+          description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: When enabled, geolocation data will be used to match the guest shopper with a relevant country-level tax zone for tax estimation on the storefront. The geolocation data will be used to determine the corresponding tax zone provided in basic tax - once the corresponding tax zone is found, the tax estimation will be applied to the store.
```

Comment:
Do we need the word data in here?

Reply:
removed 👍

Reply:
> geolocation will be used

Doesn't sound quite right to me though 🤔 Do we need a more complete rewording? e.g. "the guest shopper's geolocation will be used to determine the relevant country-level tax zone for storefront tax estimation".

I also wonder if `geolocation` may suggest a complete geolocation, something with a higher accuracy than we are offering, whereas `geolocation data` is more vague and reflects that we're only using a subset of the data.

What do you think @benpratt77?

Reply:
Perhaps a more complete rewording. data is so vague. 
So what are we actually using, we are using their IP to determine their country. With the idea that some time in the future we may also use subdivision. (I don't think we will ever try for postal code as that is entirely too inaccurate from my previous digging)

Reply:
@benpratt77 Are you satisfied with my rewording suggestion, or do you have another one to offer?

Reply:
I am satisfied with your rewording.